app/_containers/Main/containers/SectionArticles/index.tsx

```diff
@@ -1 +1,31 @@
-export default function SectionArticles() {}
+
+import cx from "clsx";
+import styles from "./index.module.scss";
+import { Post } from "@/modules/business-types";
+import CardArticle from "@/app/articles/_containers/Main/containers/CardArticle";
+
+type Props = {
+    className?: string;
+    style?: React.CSSProperties;
+    posts: Post[];
+  };
+
+export default function SectionArticles({ className, style, posts }: Props) {
+
+    return (
+        <section className={cx(styles.container, className)} style={style}>
+            <div className={styles.container}>
+                <h1>Latest Articles</h1>
```

Comment:
This must be `h2`.

Imagine the webpage is like a document. The headings (`h1`-`h6`) must form a reasonable table of content. And there should be only one `h1`.